accord-core/src/main/java/accord/local/CommandStore.java

```diff
@@ -892,28 +1057,28 @@ public final DataStore unsafeGetDataStore()
 
     final synchronized void markSafeToRead(Timestamp forBootstrapAt, Timestamp at, Ranges ranges)
     {
-        execute((PreLoadContext.Empty) () -> "Mark Safe To Read", safeStore -> {
+        execute((Empty) () -> "Mark Safe To Read", safeStore -> {
             // TODO (required): handle weird edge cases like newer at having a lower HLC than prior existing at, but higher epoch
             Ranges validatedSafeToRead = redundantBefore.validateSafeToRead(forBootstrapAt, ranges);
             safeStore.setSafeToRead(purgeAndInsert(safeToRead, at, validatedSafeToRead));
             updateMaxConflicts(ranges, at);
         }, agent);
     }
 
-    public static ImmutableSortedMap<TxnId, Ranges> bootstrap(TxnId at, Ranges ranges, NavigableMap<TxnId, Ranges> bootstrappedAt)
+    public static ImmutableSortedMap<TxnId, Ranges> bootstrap(TxnId at, Ranges ranges, NavigableMap<TxnId, Ranges> readyAt)
     {
         Invariants.requireArgument(!ranges.isEmpty());
         if (at == TxnId.NONE)
         {
-            for (Ranges rs : bootstrappedAt.values())
+            for (Ranges rs : readyAt.values())
                 Invariants.require(!ranges.intersects(rs));
         }
         else
         {
-            bootstrappedAt.floorEntry(at).getValue().containsAll(ranges);
+            readyAt.floorEntry(at).getValue().containsAll(ranges);
```

Comment:
I understand this patch only has renamed the field, but this probably should be an assert